.changeset/afraid-rice-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': minor
+---
+
+Added various new icons from Figma to the library.
```

Comment:
Is this fine? Otherwise I can also spell out each con but there's a good amount of them

Reply:
I would list them individually so developers can search the changelog to determine when a specific icon was added.

Reply:
Sorry, completely missed your comment here! Added in 6166de132097096c683d01b3d562aeaa69bf5096